common/src/main/java/dev/schmarrn/lighty/event/KeyBind.java

```diff
@@ -49,7 +49,7 @@ public static void handleKeyBind(Minecraft client) {
             ModeLoader.toggle();
         }
         if (newKeyState && !KeyBind.oldKeyState) {
-            client.setScreen(new ModeSwitcherScreen());
+            client.setScreen(new SettingsScreen());
```

Comment:
```suggestion
            client.setScreen(new ModeSwitcherScreen());
```

This has to be ModeSwitcherScreen, not SettingsScreen

Reply:
Are you sure? Right now, both this and the ModMenu settings start on the SettingsScreen. Changing this would ruin the parity between the two.

Reply:
Yes, I am sure, the ModMenu settings screen should be SettingsScreen, while the Keybind still opens ModeSwitcherScreen.

Eventually in 1.20.5/1.20.6 after porting this PR to those versions both will point to the same thing, but in 1.20.3/1.20.4 I'd prefer if they are still different because of internal technical reasons.

Reply:
Alright, it's been done.